### DISCUSSION TITLE
Edit the source code in-place while reviewing the comments of a PR

### DISCUSSION THREAD

**Discussion by sadtab** [pwntester/octo.nvim#319] [Ideas]
This plug is super dope as f*, would it be even better if you can edit the source code in place while addressing the PR comments? It will truly give a huge advantage to its user to edit the source code, accept a suggestion in place

**Comment by vitorf7** [comment]
I agree. I was looking for any suggestions that this might be implemented as when you are reviewing, if you try to go into insert mode or anything like that it says that the file is not modifiable plus none of the LSP stuff works either.
Not sure if this is something that the plugin author wants to implement or not. I know there is another plugin called `gh.nvim` that does that however in this case I prefer the way that octo does everything else in terms of viewing the code, the way you can almost type the commands in a normal language like format such as `:Octo pr create`, etc.

**Comment by pwntester** [comment]
Have you tried using the [`use_local_fs` config option](https://github.com/pwntester/octo.nvim/pull/349), that should use the files from your file system which should allow things like using an LSP on them

**Reply by vitorf7** [reply]
@pwntester Ahh I did not pay attention to that option in the documentation. Thank you so much, it is working now
